cmd/wstail/forwarders.go

```diff
@@ -75,11 +82,30 @@ func forwarderFilename(forwarder logForwarder) string {
 }
 
 // forwarderConfig returns log forwarder rsyslog config
-func forwarderConfig(forwarder logForwarder) []byte {
-	if forwarder.Spec.Protocol == "udp" {
-		return []byte(fmt.Sprintf("*.* @%v", forwarder.Spec.Address))
+func forwarderConfig(forwarder logForwarder) ([]byte, error) {
+	host, port := splitHostPort(forwarder.Spec.Address)
+	var buf bytes.Buffer
+	var config = struct {
```

Comment:
Why not pass a map instead of defining a struct?

```go
err := configTemplate.Execute(&buf, map[string]string{
  "Target": host,
  "Port": port,
  "Protocol": forwarder.Spec.Protocol,
})
```